lib/erblint-github/linters/custom_helpers.rb

```diff
@@ -64,6 +64,13 @@ def generate_offense(klass, processed_source, tag, message = nil, replacement =
         add_offense(processed_source.to_source_range(tag.loc), offense, replacement)
       end
 
+      def generate_offense_from_source_range(klass, source_range, message = nil, replacement = nil)
+        message ||= klass::MESSAGE
+        message += "\nLearn more at https://github.com/github/erblint-github#rules.\n"
+        offense = ["#{simple_class_name}:#{message}", source_range.source].join("\n")
+        add_offense(source_range, offense, replacement)
```

Comment:
This helper allows the output of the ERB lint message to be something like:

```
ERBLint::Linters::A11yAvoidGenericLinkTextCounter:Avoid using generic link text such as Read more, Learn more, Click here, More, Link.
<a href="github.com/about">Learn more
In file: app/views/test.html.erb:46
```
where we can specify a range of the code source to be output. In this case we include the first anchor tag and the text content for ease of identification.

The other helper we commonly use, `generate_offense` only allows a single tag to be output in the message.